third_party/3/maxminddb/reader.pyi

```diff
@@ -0,0 +1,32 @@
+# Stubs for maxminddb.reader (Python 3)
+
+from ipaddress import IPv4Address, IPv6Address
+
+from types import TracebackType
+from typing import Any, Mapping, Optional, Sequence, Type, Union
+
+class Reader:
+    closed: bool = ...
+    def __init__(self, database: bytes, mode: int = ...) -> None: ...
+    def metadata(self) -> 'Metadata': ...
+    def get(self, ip_address: Union[IPv4Address, IPv6Address]) -> Any: ...
```

Comment:
`ip_address` must be a `str`. You could also return `Optional[Any]`, since `None` is a potential return type. This forces users to ensure that `None` was not returned.